packages/data-stores/.eslintrc.js

```diff
@@ -5,4 +5,15 @@ module.exports = {
 	rules: {
 		'import/no-extraneous-dependencies': [ 'error', { packageDir: __dirname } ],
 	},
+	overrides: [
+		{
+			files: [ '**/test/**/*' ],
+			rules: {
+				'import/no-extraneous-dependencies': [
+					'error',
+					{ packageDir: [ __dirname, __dirname + '/../..' ] },
```

Comment:
We want the project root here? Let's use `path` to get us there. Paths parts like `/../..` always end up being problematic for someone 🙂 

```suggestion
					{ packageDir: [ __dirname, path.join( __dirname, '..', '..' ) ] },
```

Related: #39088

Reply:
Fair. I'm an ex-windows user and I always appreciated OSS trying to keep their code agnostic :D